src/features/user/userSlice.js

```diff
@@ -0,0 +1,8 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+
+
+const initialState ={
```

Comment:
what do we intend using this for?

Reply:
To get state of the app and hold users details